src/Shared/Microsoft.DotNet.Services.Utility/GitHelpers.cs

```diff
@@ -3,6 +3,7 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Linq;
```

Comment:
Nit: I think VS added this.

Reply:
No, I was trying to add something to it and then it didn't work and I forgot to undo all my changes.